--- external/src/swehel.c	2018-01-26 13:23:54.000000000 +0100
+++ src/libswe/swehel.c	2018-08-17 21:46:54.352921729 +0200
@@ -3250,8 +3250,8 @@
   int32 epheflag = helflag & (SEFLG_JPLEPH|SEFLG_SWIEPH|SEFLG_MOSEPH);
   dret[0] = tjdstart; /* will be returned in error case */
   if (TypeEvent == 1 || TypeEvent == 2) {
-    if (serr != NULL)
-      strcpy(serr, "error: the moon has no morning first or evening last");
+    if (serr_ret != NULL)
+      strcpy(serr_ret, "error: the moon has no morning first or evening last");
     return ERR;
   }
   strcpy(ObjectName, "moon");
