--- external/src/swehel.c	2018-01-26 13:23:54.000000000 +0100
+++ src/libswe/swehel.c	2018-08-17 21:54:07.039409588 +0200
@@ -228,7 +228,7 @@
   double OpticDia = dobs[4];
   double OpticTrans = dobs[5];
   AS_BOOL is_scotopic = FALSE;
-  JDNDaysUT = JDNDaysUT; /* currently not used, statement prevents compiler warning */
+  JDNDaysUT += 0.0; /* currently not used, statement prevents compiler warning */
   SNi = SN;
   if (SNi <= 0.00000001) SNi = 0.00000001;
   /* 23 jaar as standard from Garstang*/
@@ -553,7 +553,7 @@
   double dut;
   static TLS double tjdlast;
   static TLS double ralast;
-  helflag = helflag; /* statement prevents compiler warning */
+  helflag += 0; /* statement prevents compiler warning */
   *serr = '\0';
   if (JDNDaysUT == tjdlast)
     return ralast;
@@ -1265,7 +1265,7 @@
 static double Bcity(double Value, double Press)
 {
   double Bcity = Value;
-  Press = Press; /* unused; statement prevents compiler warning */
+  Press += 0.0; /* unused; statement prevents compiler warning */
   Bcity = mymax(Bcity, 0);
   return Bcity;
 }
