--- external/src/sweph.c	2018-01-26 13:46:06.000000000 +0100
+++ src/libswe/sweph.c	2018-08-17 23:36:58.300331476 +0200
@@ -256,6 +256,7 @@
   *s = '\0';
 #if !defined(__APPLE) 
   len = AS_MAXCH;
+  bytes = 0;
 #if MSDOS
   bytes = GetModuleFileName(dllhandle, (TCHAR*) s, len);
 #else
@@ -275,9 +276,7 @@
     bytes = readlink("/proc/self/exe", s, len);
   #endif
 #endif
-  if(bytes >= 0) {
-    s[bytes] = '\0';
-  }
+  s[bytes] = '\0';
 #endif
   return s;
 }
