--- external/src/sweph.c	2018-12-21 00:11:47.040930414 +0100
+++ src/libswe/sweph.c	2019-01-10 23:13:16.395002324 +0100
@@ -7368,9 +7360,10 @@
     }
     // search string is traditional name
     *sp = '\0';	/* cut off after first field to get star name, ',' -> '\0' */
-    strncpy(fstar, s, SE_MAX_STNAME);
+    size_t length = strnlen(s, SE_MAX_STNAME);
+    memcpy(fstar, s, length);
+    fstar[length] = '\0';  /* force termination */
     *sp = ',';  /* add comma again */
-    fstar[SE_MAX_STNAME] = '\0';	/* force termination */
     // remove white spaces from star name
     while ((sp = strchr(fstar, ' ')) != NULL)
       swi_strcpy(sp, sp+1);
