diff --git a/src/libswe/swehouse.c b/src/libswe/swehouse.c
index 627796a..551d26b 100644
--- a/src/libswe/swehouse.c
+++ b/src/libswe/swehouse.c
@@ -1810,7 +1810,7 @@ double CALL_CONV swe_house_pos(
   //double demc;
   double fh, ra0, tanfi, fac, dfac, tanx;
   double x[3], xasc[3], raep, raaz, oblaz, xtemp; /* BK 21.02.2006 */
-  double hcusp[36], ascmc[10];
+  double hcusp[37], ascmc[10];
   double sine = sind(eps);
   double cose = cosd(eps);
   double c1, c2, d, hsize;
