diff --git a/src/libswe/swecl.c b/src/libswe/swecl.c
index 81a57f3..2e60c22 100644
--- a/src/libswe/swecl.c
+++ b/src/libswe/swecl.c
@@ -3007,8 +3007,8 @@ void CALL_CONV swe_set_lapse_rate(double lapse_rate)
  *                      * geometric horizon = plane perpendicular to gravity *
  * double geoalt;       * altitude of observer above sea level in meters *
  * double atpress;      * millibars (hectopascal) *
- * double lapse_rate;    * (dT/dh) [deg K/m]
  * double attemp;       * degrees C *
+ * double lapse_rate;    * (dT/dh) [deg K/m]
  * int32  calc_flag;    * either SE_APP_TO_TRUE or
  *                      *        SE_TRUE_TO_APP
  *
@@ -3161,7 +3161,7 @@ static double calc_dip(double geoalt, double atpress, double attemp, double laps
 {
   /* below formula is based on A. Thom, Megalithic lunar observations, 1973 (page 32).
   * conversion to metric has been done by
-  * V. Reijs, 2000, http://www.iol.ie/~geniet/eng/refract.htm
+  * V. Reijs, 2000, http://archaeocosmology.org/eng/refract.htm
   */
   double krefr = (0.0342 + lapse_rate) / (0.154 * 0.0238);
   double d = 1-1.8480*krefr*atpress/(273.16+attemp)/(273.16+attemp);
