diff --git a/src/libswe/sweph.c b/src/libswe/sweph.c
index 78779cc..bb7a0da 100644
--- a/src/libswe/sweph.c
+++ b/src/libswe/sweph.c
@@ -2178,7 +2178,7 @@ static int jplplan(double tjd, int ipli, int32 iflag, AS_BOOL do_save,
 static int sweph(double tjd, int ipli, int ifno, int32 iflag, double *xsunb, AS_BOOL do_save, double *xpret, char *serr)
 {
   int i, ipl, retc, subdirlen;
-  char s[AS_MAXCH], subdirnam[AS_MAXCH], fname[AS_MAXCH], *sp;
+  char s[2 * AS_MAXCH], subdirnam[AS_MAXCH], fname[AS_MAXCH], *sp;
   double t, tsv;       
   double xemb[6], xx[6], *xp;
   struct plan_data *pdp;
